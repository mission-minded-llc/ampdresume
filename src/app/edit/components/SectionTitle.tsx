import { Box, Divider, Typography } from "@mui/material";

export const SectionTitle = ({ title }: { title: string }) => (
  <>
    <Divider sx={{ mb: 2 }} />
    <Box>
      <Typography variant="h5" component="h1">
        {title}
      </Typography>
    </Box>
    <Divider sx={{ mb: 4, mt: 2 }} />
  </>
);
