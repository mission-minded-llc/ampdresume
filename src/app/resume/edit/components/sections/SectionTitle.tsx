import { Box, Divider, Typography } from "@mui/material";

export const SectionTitle = ({ title }: { title: string }) => (
  <>
    <Divider sx={{ mb: 2 }} />
    <Box>
      <Typography variant="h5" sx={{ mb: 2 }}>
        {title}
      </Typography>
    </Box>
    <Divider sx={{ mb: 4, mt: 2 }} />
  </>
);
