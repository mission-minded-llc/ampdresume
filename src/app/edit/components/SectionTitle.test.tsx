import "@testing-library/jest-dom";

import React from "react";
import { SectionTitle } from "./SectionTitle";
import { render } from "@testing-library/react";

describe("SectionTitle", () => {
  it("renders correctly with the provided title", () => {
    const { getByText, container } = render(<SectionTitle title="Test Title" />);
    expect(container).toMatchSnapshot();

    expect(getByText("Test Title")).toBeInTheDocument();
    expect(getByText("Test Title").tagName).toBe("H1");
  });
});
