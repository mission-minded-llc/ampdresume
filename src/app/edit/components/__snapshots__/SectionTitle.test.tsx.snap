// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionTitle renders correctly with the provided title 1`] = `
<div>
  <hr
    class="MuiDivider-root MuiDivider-fullWidth css-1nih0dr-MuiDivider-root"
  />
  <div
    class="MuiBox-root css-0"
  >
    <h1
      class="MuiTypography-root MuiTypography-h5 css-16xl4zq-MuiTypography-root"
    >
      Test Title
    </h1>
  </div>
  <hr
    class="MuiDivider-root MuiDivider-fullWidth css-1o0cmko-MuiDivider-root"
  />
</div>
`;
